roles/openshift_logging/templates/elasticsearch.yml.j2

```diff
@@ -32,7 +32,7 @@ discovery:
 
 gateway:
   expected_master_nodes: ${NODE_QUORUM}
```

Comment:
I think using environment variables will be best as we can then maintain a single configmap and have the deployment config (and eventually statefulset) dictate what should be used.  So there wouldn't be a change